app/config/permissions.js

```diff
@@ -287,6 +287,10 @@ const groups = [
       ],
       full: [
         'manage-signatures',
+        'manage-only-specific-signatures',
+        'send-only-specific-cases-to-vp',
+        'create-submissions',
```

Comment:
needs graph access to submissions

Reply:
https://github.com/kanselarij-vlaanderen/app-kaleidos/pull/499

Will also update the new mu-auth PR

Reply:
Apparently @sergiofenoll is clairvoyant, because it's already in the new mu-auth ;-) https://github.com/kanselarij-vlaanderen/app-kaleidos/pull/469/files#diff-c0623dcb5422872d71dd79b855fe9a11e8c09c6b585c53799c5a0f9ef54639dfR522